packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendar.tsx

```diff
@@ -74,6 +74,16 @@ export type DatepickerCalendarStylesProps = never;
 
 export const datepickerCalendarClassName = 'ui-datepicker__calendar';
 
+const getRidOfSecondsMinutesHours = (date: Date): Date => {
+  return new Date(date.getFullYear(), date.getMonth(), date.getDate());
+};
+
+const normalizeDateInGrid = (date: Date): Date => {
+  const result = new Date(date.getTime());
+  result.setDate(1);
+  return getRidOfSecondsMinutesHours(result);
+};
+
```

Comment:
Should we move that to `date-time-utilities` so it can be shared if needed?

Reply:
I realized that I managed to implement some things differently and thus the method is no longer needed.